packages/x-charts/src/internals/store/useCharts.ts

```diff
@@ -63,7 +63,7 @@ export function useCharts<
   const innerChartRootRef = React.useRef<HTMLDivElement>(null);
   const innerSvgRef = React.useRef<SVGSVGElement>(null);
 
-  const storeRef = React.useRef<ChartStore<TSignaturesWithCorePluginSignatures> | null>(null);
+  const storeRef = React.useRef<Store<ChartState<TSignaturesWithCorePluginSignatures>>>(null);
```

Comment:
This is subjective, but I think keeping the `ChartsStore<TSignature>` is easier to read and write.